translate/storage/lisa.py

```diff
@@ -164,15 +173,7 @@ def settarget(self, text, lang='xx', append=False):
                         pass
                 languageNode.text = text
         else:
-            self.set_target_dom(None, False)
-
-    def gettarget(self, lang=None):
-        """retrieves the "target" text (second entry), or the entry in the
-        specified language, if it exists"""
-        return self.getNodeText(self.get_target_dom(lang),
-                                getXMLspace(self.xmlelement,
-                                            self._default_xml_space))
-    target = property(gettarget, settarget)
```

Comment:
IMHO `property(getter, setter)` syntax is an aberration that we should try to get rid of. So lets replace that with the new properties syntax.

As I see we should keep both `gettarget` and the properties (using new syntax), since in this particular scenario we cannot blindly replace the functions with the properties. To my understanding the properties don't allow providing the `lang`, but as you say since we cannot be sure if any third party is using the functions by passing that `lang`, so we cannot drop them.

This is my first impression, and I think that addressing this this way is safe. Eventually I would like to remove those functions, but I am not sure if they provide some key functionality.